services/movie.py

```diff
@@ -23,18 +26,19 @@ def get_movie_by_id(movie_id: int) -> Movie:
 
 
 def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None,
```

Comment:
The type annotations for `genres_ids` and `actors_ids` should be more specific. Consider using `list[int]` instead of `list` to indicate that these are lists of integers.